### PR TITLE
Conan quick start

### PR DESCRIPTION
**Pull request by alfred-bratterud**
I've added a suggested quick start guide for the service- and kernel workflows to the top of the readme now. I haven't updated the rest of the readme yet, but I'm wondering if some of @staiyeba's details should be moved to the external docs or if we want to keep them here. @MagnusS let me know what you think. @KristianJerpetjon great if you can verify that the quick-start guide is correct and also works for linux distros.

### CODE REVIEW

**Comment by MagnusS** [comment]
@alfred-bratterud plan was to get the full process into the READMEs first as its getting finalised (with at least service + kernel development workflow), then move portions of it into external docs and refer to them here. The external docs are a bit outdated at the moment, so they will also need some work before we have somewhere to move the text.

File: README.md [hunk 1]
```diff
@@ -16,6 +11,66 @@ IncludeOS is free software, with "no warranties or restrictions of any kind".
...
+The above shows how to build bootable IncludeOS binaries using pre-built versions the IncludeOS libraries. In order to modify IncludeOS itself or contribute to IncludeOS development, you need a different workflow. Start by cloning IncludeOS.
+
+```
+$ git clone https://github.com/includeos/IncludeOS.git
+```
+
+Now create a conan layout file, or edit your own copy of `layout.txt`. Have the first line point to `your_kernel_build_dir`, e.g. where you want the compiled IncludeOS libraries to be built while you work. You can now tell conan to use this local build instead of the prebuilt binary versions.
+
+```
+$ conan editable add your_IncludeOS_fork -l your_layout.txt includeos/0.15.0@includeos/test
```
[the remark this answers is not in the document]
Comment (MagnusS) [reply README.md]: The instructions already in the readme further down works, looks like you missed commenting out "version" in conanfile.py.

File: README.md [hunk 2]
```diff
@@ -16,6 +11,66 @@ IncludeOS is free software, with "no warranties or restrictions of any kind".
...
+## Quick start
+### Install the dependencies for building IncludeOS services:
+* [The conan package manager](https://docs.conan.io/en/latest/installation.html)
+* CMake
+* clang, or alternatively gcc on linux
+
+Now install the conan profiles and remotes for getting binary packages of includeos libraries.
+```
+$ conan install https://github.com/includeos/conan_config.git
+```
```
[the remark this answers is not in the document]
Comment (alfred-bratterud) [reply README.md]: that's unfortunate. I fixed the command at least.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (alfred-bratterud) [reply README.md]: Yea @KristianJerpetjon has to weight in on this, it looks like we have to be super exact since we're on v0.x?

**Comment by alfred-bratterud** [comment]
@MagnusS / @staiyeba I'll leave the rest of this to you guys. I think we should have a very quick quick-start at the beginning, at least for the service development workflow. I don't mind if the kernel workflow is further down, maybe mine can just be removed, but I think it should have more focus on getting started with IncludeOS rather than on teaching people conan. Up to you now, I won't make further changes.

**Comment by staiyeba** [comment]
I agree the quick start should be really simple and now that we have had more time to work on read the docs and are getting ready for a release I think @alfreds Quick start is the way to go :)

**Pull request merged**